cloudformation/cf10aws.yml

```
@@ -0,0 +1,365 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters: 
+
+# MyIP,create
+  MyIP:
+    Description: IP address allowed to access EC2
+    Type: String
+
+# KeyName,create
+  KeyName:
+    Description: The EC2 Key Pair to allow SSH access to the instance
+    Type: "AWS::EC2::KeyPair::KeyName"
+
+Resources: 
+
+# Vpc,create
+  Vpc10:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.10.0.0/20
+      EnableDnsSupport: true
+      Tags:
+        - Key: Name
+          Value: Vpc10Cf
+
+# InternetGateWay,create
+  InternetGateWay10:
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: InternetGateWay10Cf
+
+  AttachGateway:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      VpcId: !Ref Vpc10
+      InternetGatewayId: !Ref InternetGateWay10
+
+# Subnet,create
+  PublicSubnet11:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.0.0/23
+      Tags:
+        - Key: Name
+          Value: PublicSubnet11Cf
+
+  PrivateSubnet11:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.2.0/23
+      Tags:
+        - Key: Name
+          Value: PrivateSubnet11Cf
+
+  PublicSubnet12:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.4.0/23
+      Tags:
+        - Key: Name
+          Value: PublicSubnet12Cf
+
+  PrivateSubnet12:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.6.0/23
+      Tags:
+        - Key: Name
+          Value: PrivateSubnet12Cf
+
+# RouteTable,RouteTableAssociation,Route,create
+  RouteTable10Pus11:
```

`Pus`はPublicSubnetの略ですかね？だとすると、以下のようになるかなと思います！
```suggestion
  RouteTable10PuS11:
```

---

何度も申し訳ありません！
修正します！